### ISSUE TITLE
Instagram banned?

### ISSUE THREAD

**Issue by Dandw99** [igdmapps/igdm#1419]
After i installed this Instagram locked my account for suspicious stuff, did this happen to any of you?

**Comment by etbigevil** [comment]
yes me too as of today! I got these error 
![IMG_1760](https://user-images.githubusercontent.com/85594141/121276035-be5cf980-c882-11eb-99d6-431400baa4bc.jpeg)

**Comment by Anzio1000** [comment]
Yes, **Instagram will block your account for using IGDM**. In other words, the days of IGDM are numbered.

**Comment by 7urkm3n** [comment]
@Anzio1000 What do you mean by `the days of IGDM are numbered`, can you explain it ?

**Comment by Anzio1000** [comment]
@7urkm3n A few weeks ago I attempted to open IGDM. I encountered the problem others have mentioned where images were not resolving. However, DMs were working via IGMD. However, Instagram sent me a notice that stated I had been logged out of Instagram (I take this as a first warning). 

A week later I tried to use IGDM, but the program would not connect at all with Instagram. I installed and uninstalled the program more than once, my license was up to date and working, yet I could not connect to my Instagram account via IGDM. Since IGDM has no support at all, as a last resort I purchased a new version of IGDM (third time I have done this).

With the new license, I was able to connect to Instagram via IGDM. However, Instagram locked me out of my account for 24 hours and send me the same warning noted at the beginning of this thread. 

Therefore, my conclusion is that Instagram is treating IGDM as malware and my research into this suggests that further connectivity to Insta via IGDM will result in increasing account closures from insta and ultimately account deletion. 

I will never connect to Insta via IGDM again.

**Comment by mayeaux** [comment]
Yeash good to know. The site should have a warning on it. I tried to use it and couldn't login seems like that was fortuitous!

**Comment by actualdankcoder** [comment]
using custom client's is against instagram tos and very detectable as well, no wonder people are being locked out

**Comment by climbTheStairs** [comment]
I had a similar issue, but I wasn't locked out of my account.

After logging in with IGdm, I was immediately logged out.  I had Instagram open in my browser too, where I got a popup "Suspicious activity detected", and was forced to verify my email and change my password.

Following that, IGdm seems to be working fine after working fine; however, does this indicate some problem with the app that might trigger Instagram's suspicion?

**Comment by 7urkm3n** [comment]
Guys, i made more simple extension. All you need is sign in browser instagram. 

https://github.com/igimapps/IGim-Extension
